src/translations/en/translation.json

```diff
@@ -105,5 +105,46 @@
   "Dinner 🍽️": "Dinner 🍽️",
   "Walk 🚶🏽": "Walk 🚶🏽",
   "Water the plants 🪴 indoors": "Water the plants 🪴 indoors",
-  "Me time 🧘🏽😌": "Me time 🧘🏽😌"
+  "Me time 🧘🏽😌": "Me time 🧘🏽😌",
+  "dontaskagain": "Don't ask again for this action?",
+  "Note": "Note",
+  "Complete goal": "Complete goal",
+  "Delete goal": "Delete goal",
+  "Share goal": "Share goal",
+  "Choose contact": "Choose contact",
+  "Collaborate on goal": "Collaborate on goal",
```

Comment:
@vinaybadgujar102 Just make these keys one word, like "share goal" => "shareGoal" and follow camel case style. rest looks good to me.

Reply:
To make the requested changes I also have to change the values of "getConfirmButtonText" object in "myGoals.ts".

Reply:
@Tushar-4781 made the required changes